.ado/publish.yml

```diff
@@ -1,45 +1,195 @@
+    displayName: Build Platform-Independent Py Packages
+    condition: eq(variables['Agent.OS'], '${{ variables.PlatfIndepPyPackBuildPlatf }}')
 
   - script: |
-      vsce publish --pre-release --pat $(PAT) 2>&1 > pub.log
+      dir target\wheels\*
+    displayName: List Py Packages on Win
+    condition: eq(variables['Agent.OS'], 'Windows_NT')
+
+  - script: |
+      ls target/wheels/*
+    displayName: List Py Packages on non-Win
+    condition: ne(variables['Agent.OS'], 'Windows_NT')
+
+  - publish: $(System.DefaultWorkingDirectory)/target/wheels
+    artifact: Wheels.Mac
+    displayName: Upload Python Artifacts Mac
+    condition: eq(variables['Agent.OS'], 'Darwin')
+
+  - publish: $(System.DefaultWorkingDirectory)/target/wheels
+    artifact: Wheels.Win
+    displayName: Upload Python Artifacts Win
+    condition: eq(variables['Agent.OS'], 'Windows_NT')
+
+  - publish: $(System.DefaultWorkingDirectory)/target/wheels
+    artifact: Wheels.Linux.${{ variables['arch'] }}
+    displayName: Upload Python Artifacts Linux
+    condition: eq(variables['Agent.OS'], 'Linux')
+
+
+- job: "Publish"
+  dependsOn: Build
+  pool:
+    vmImage: 'windows-latest'
+
+  steps:
+  - download: current
+    artifact: Wheels.Win
+    displayName: Download Python Artifacts Win
+
+  - download: current
+    artifact: Wheels.Mac
+    displayName: Download Python Artifacts Mac
+
+  - download: current
+    artifact: Wheels.Linux.x86_64
+    displayName: Download Python Artifacts Linux
+
+  - download: current
+    artifact: Wheels.Linux.aarch64
+    displayName: Download Python Artifacts Linux
+
+  - script: |
+      mkdir                               target\wheels
+      move ..\Wheels.Linux.x86_64\*.whl   target\wheels
+      move ..\Wheels.Linux.aarch64\*.whl  target\wheels
+      move ..\Wheels.Win\*.whl            target\wheels
+      move ..\Wheels.Mac\*.whl            target\wheels
+      dir                                 target\wheels\*
+    displayName: Move Py Artifacts to Publishing Dir
+
+  - download: current
+    artifact: VSIX
+    displayName: Download VSCode Extension Artifact
+
+  - script: |
+      npm install -g @vscode/vsce
+    displayName: Install Prereqs for VSCode Ext Publishing
+
+
+  # NB: Publishing must be the last thing (done only if all previous steps are successful).
+
+  - task: EsrpRelease@2
+    inputs:
+     ConnectedServiceName: 'ESRP_Release'
+     Intent: 'PackageDistribution'
+     ContentType: 'PyPi'
+     PackageLocation: '$(System.DefaultWorkingDirectory)/target/wheels'
+     Owners: '$(OwnerPersonalAlias)@microsoft.com'  # NB: ESRP corrected from group email `qdevtools` to personal one.
+                                                    #     Group email here fails the task with non-actionable output.
+     Approvers: 'billti@microsoft.com'
+     # Auto-inserted Debugging defaults:
+     ServiceEndpointUrl: 'https://api.esrp.microsoft.com'
+     MainPublisher: 'QuantumDevelpmentKit'     # ESRP Team's Correction.
```

Comment:
Just to be sure, does the comment mean that the typo is intentional?

Reply:
I have just noticed that "Deve**lp**ment" is written with a typo. Good eye! :-D
The default was completely unrelated to 'QuantumDevelopmentKit', so I believe the typo was not intentional. I will test to make sure.

Reply:
Confirmed, the typo is mandatory :-D, I added the comment to the [file](https://github.com/microsoft/qsharp/pull/466/files#diff-8cc88adda4da183aafc286681a96e285509fd92919e0549a163c376f2ee12994), see below.
The ESRP Team's contact keeps silent after I asked to comment about that.
```yml
MainPublisher: 'QuantumDevelpmentKit'          # ESRP Team's Correction (including the critical typo "Develpm").
```